united.py

```diff
@@ -1,25 +1,51 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
+import calendar
 import csv
 from datetime import datetime
 import os
 import argparse
+from wsgiref import headers
+from xml.etree import ElementTree
 
-from flask.ext.sqlalchemy import SQLAlchemy
+import re
+from flask_sqlalchemy import SQLAlchemy
```

Comment:
So I think you also need to change the `Flask-SQLAlchemy` version in `requirements.txt` to 2.1.